modules/add.js

```diff
@@ -14,10 +17,26 @@ const populateMeals = async () => {
           <i class="fas fa-heart like-icon"></i>
         </div>
         <span></span>
-        <button class="comments">Comments</button>
         <button class="reservations">Reservations</button>
     `;
     listItems.appendChild(listItem);
+    listItem.appendChild(commentBtn);
+    commentBtn.addEventListener('click', () => {
+      const commentWindow = document.querySelector('.comment-window');
+      const closeBtn = document.createElement('i');
+      closeBtn.className = 'fa-solid fa-x';
+      commentWindow.innerHTML = `
+      <img src=${meal.strCategoryThumb} alt="Meal-image" class="comment-meal-image">
+      <div class="comment-meal-info">
+        <h2 class='window-title'>${meal.strCategory}</h2>
+        <p class='window-description'>${meal.strCategoryDescription}</p>
+      </div>`;
+      commentWindow.classList.toggle('show-comment-window');
+      commentWindow.appendChild(closeBtn);
+      closeBtn.addEventListener('click', () => {
+        commentWindow.classList.toggle('show-comment-window');
+      });
+    });
```

Comment:
It's a nice idea to add an event listener to the comment button as it is created. However, you would have to add more code to position it above the reservations button which has been created already. 
I suggest you create an event listener on the document and target the comments button like so
`document.addEventListener('click', (e) => { if (e.target.className === 'comments') { }}` and call the comment modal function to create the modal.

Reply:
Thank you very much. I applied this method to code and it works now 💯